gef.py

```diff
@@ -9558,16 +9566,16 @@ def infos(self, address: int) -> None:
 
         if sect:
             gef_print(f"Page: {format_address(sect.page_start)} {RIGHT_ARROW} "
-                      f"{format_address(sect.page_end)} (size={sect.page_end-sect.page_start:#x})")
-            gef_print(f"Permissions: {sect.permission}")
-            gef_print(f"Pathname: {sect.path}")
-            gef_print(f"Offset (from page): {addr.value-sect.page_start:#x}")
-            gef_print(f"Inode: {sect.inode}")
+                      f"{format_address(sect.page_end)} (size={sect.page_end-sect.page_start:#x})"
+                      f"\nPermissions: {sect.permission}"
+                      f"\nPathname: {sect.path}"
+                      f"\nOffset (from page): {addr.value-sect.page_start:#x}"
+                      f"\nInode: {sect.inode}")
```

Comment:
and here?

Reply:
in this particular case I think I want to avoid mixing 'args' and concatenated strings across lines